### ISSUE TITLE
Sentence segmentation doesn't match parse

### ISSUE THREAD

**Issue by capdevc** [honnibal/spaCy#184]
Hi. I'm using the latest 0.99 installed via conda with the latest models downloaded

I'm having trouble with sentence segmentation.

``` python
In[39]: text = "I look forward to using Thingamajig.  I've been told it will make my life easier..."
In[40]: t = SPACY(text)
In[41]: list(t.sents)
Out[44]: [I look forward to using Thingamajig.  I've been told it will make my life easier...]
In[45]: len(list(t.sents))
Out[45]: 1
```
Here it's claiming that there is only one sentence. This is even though the parse clearly shows two:
``` python
In[47]: [(tok.orth_, tok.head ) for tok in t]
Out[48]: 
[('I', look ),
 ('look', look ),   <----- ROOT
 ('forward', look ),
 ('to', look ),
 ('using', to ),
 ('Thingamajig', using ),
 ('.', look ),
 (' ', told ),      <---- problem?
 ('I', told ),
 ("'ve", told ),
 ('been', told ),
 ('told', told ),       <----- ROOT
 ('it', make ),
 ('will', make ),
 ('make', told ),
 ('my', life ),
 ('life', easier),
 ('easier', make ),
 ('...', told )]
```

This may be related to #154. If I remove the extra space between the two sentences in the original text (there are two between "Thingamajig." and "I've") everything works correctly and I get two sentences.

``` python
In[53]: text = "I look forward to using Thingamajig. I've been told it will make my life easier..."
In[54]: t = SPACY(text)
In[55]: len(list(t.sents))
Out[55]: 2
In[56]: [(tok.orth_, tok.head) for tok in t]
Out[56]: 
[('I', look ),
 ('look', look ),
 ('forward', look ),
 ('to', look ),
 ('using', to ),
 ('Thingamajig', using ),
 ('.', look ),
 ('I', told ),
 ("'ve", told ),
 ('been', told ),
 ('told', told ),
 ('it', make ),
 ('will', make ),
 ('make', told ),
 ('my', life ),
 ('life', easier),
 ('easier', make ),
 ('...', told )]
In[57]: list(t.sents)
Out[57]: 
[I look forward to using Thingamajig.,
 I've been told it will make my life easier...]
```

Thanks for the wonderful software!

**Comment by honnibal** [comment]
I think I know how I broke this. Thanks, will add the test and fix this.

**Comment by honnibal** [comment]
This should be fixed in v0.100. Please reopen this if it's not.
